context-v1.jsonld

```diff
@@ -16,6 +12,11 @@
       "@id": "sidetree:operation",
       "@type": "@id",
       "@container": "@set"
+    },
+    "JsonWebKey2020": {
```

Comment:
it should look like this: https://github.com/w3c/did-core-registries/blob/master/contexts/did-v0.11.jsonld#L14